yaml.go

```diff
@@ -1,27 +1,52 @@
 package manifestival
 
 import (
+	"errors"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"net/url"
 	"os"
 	"path"
+	"strings"
 
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/util/yaml"
 )
 
 // Parse parses YAML files into Unstructured objects.
 //
-// It supports 4 cases today:
+// It supports 5 cases today:
 // 1. pathname = path to a file --> parses that file.
 // 2. pathname = path to a directory, recursive = false --> parses all files in
 //    that directory.
 // 3. pathname = path to a directory, recursive = true --> parses all files in
 //    that directory and it's descendants
 // 4. pathname = url --> fetches the contents of that URL and parses them as YAML.
+// 5. pathname = combination of all previous cases, the string can contain
+//    multiple records (file, directory or url) separated by comma
 func Parse(pathname string, recursive bool) ([]unstructured.Unstructured, error) {
+
+	pathnames := strings.Split(pathname, ",")
+	aggregated := []unstructured.Unstructured{}
+	for _, pth := range pathnames {
+		if pth == "" {
+			return nil, errors.New("input contains an empty path")
+		}
```

Comment:
What would happen if you just removed this block entirely? Missing file error? I think i'd prefer that, since that's effectively what it is.

Reply:
You are right, gonna fix that, thx